boilerplate/package.json

```diff
@@ -56,8 +56,9 @@
     "react-native": "0.74.3",
     "react-native-drawer-layout": "4.0.0-alpha.9",
     "react-native-gesture-handler": "~2.16.1",
+    "react-native-keyboard-controller": "^1.12.7",
     "react-native-mmkv": "^2.12.2",
-    "react-native-reanimated": "~3.10.1",
+    "react-native-reanimated": "~3.15.0",
```

Comment:
Upgrade `react-native-reanimated` to fix [the issue](https://github.com/infinitered/ignite/pull/2722#issuecomment-2272362409) Lizzi reported.